image.go

```diff
@@ -0,0 +1,9 @@
+package containerd
+
+import "github.com/containerd/containerd/images"
+
+type Image struct {
+	client *Client
+
+	i images.Image
+}
```

Comment:
It would be very helpful, if we could add some frequently used function here, `GetManifest`, `GetConfig`, `GetSize` etc.